src/chrome/chromeDebugAdapter.ts

```diff
@@ -472,6 +481,60 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
         } else if (notification.hitBreakpoints && notification.hitBreakpoints.length) {
             reason = 'breakpoint';
 
+            const hitBreakpoints = notification.hitBreakpoints;
+            let allStopOnEntryBreakpoints = true;
+
+            // If there is a breakpoint which is not a stopOnEntry breakpoint, we appear as if we hit that one
+            // This is particularly done for cases when we end up with a user breakpoint and a stopOnEntry breakpoint on the same line
+            hitBreakpoints.forEach(bp => {
+                if (!this._stopOnEntryBreakpointIdToRequestedFileName.has(bp)) {
+                    notification.hitBreakpoints = [bp];
+                    allStopOnEntryBreakpoints = false;
+                }
+            });
+
+            // If all the breakpoints on this point are stopOnEntry breakpoints
+            // This will be true in cases where it's a single breakpoint and it's a stopOnEntry breakpoint
+            // This can also be true when we have multiple breakpoints and all of them are stopOnEntry breakpoints, for example in cases like index.js and index.bin.js
+
+            if (allStopOnEntryBreakpoints) {
+                // Get the mapped url of the file where execution is paused
+                const pausedScriptId = notification.callFrames[0].location.scriptId;
+                const pausedScriptUrl = this._scriptsById.get(pausedScriptId).url;
+
+                const mappedUrl = this._pathTransformer.scriptParsed(pausedScriptUrl);
+                let normalizedMappedUrl: string;
+                if (mappedUrl !== undefined) {
+                    normalizedMappedUrl = path.normalize(mappedUrl);
+                }
+
+                // If the file has unbound breakpoints, make sure to resolve them first and then decide to continue or not
+                if (this._pendingBreakpointsByUrl.has(normalizedMappedUrl)) {
+                    this.resolvePendingBreakpoint(this._pendingBreakpointsByUrl.get(normalizedMappedUrl))
```

Comment:
Normalizing paths is tricky. Could you add a message that says: We normalized AbC to abc, and we did (or didn't) find any breakpoints for that url? I think that could prove very useful in case we run into any issues